cmd/cosign/cli/options/verify.go

```diff
@@ -141,6 +141,8 @@ type VerifyBlobOptions struct {
 	Rekor               RekorOptions
 	Registry            RegistryOptions
 	CommonVerifyOptions CommonVerifyOptions
+
+	RFC3161TimestampPath string
```

Comment:
we could add this to `CommonVerifyOptions` alongside `TSACertChainPath`

Reply:
@priyawadhwa RFC3161TimestampPath is only used by blob verification, I am not sure it follows the mission of common verify options.

Reply:
ah gotcha yah let's leave it then, thanks!